libs/agno/tests/integration/knowledge/test_arxiv_knowledge_base.py

```diff
@@ -0,0 +1,305 @@
+import uuid
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+from agno.document import Document
+from agno.document.reader.arxiv_reader import ArxivReader
+from agno.knowledge.arxiv import ArxivKnowledgeBase
+from agno.vectordb.lancedb import LanceDb
+
+
+@pytest.fixture
+def setup_vector_db():
+    """Setup a temporary vector DB for testing."""
+    table_name = f"arxiv_test_{uuid.uuid4().hex[:8]}"
+    vector_db = LanceDb(table_name=table_name, uri="tmp/lancedb")
+    yield vector_db
+    # Clean up after test
+    vector_db.drop()
+
+
+@pytest.fixture
+def mock_arxiv_documents():
```

Comment:
We can't use mock documents, that is kind of against the idea of the integration test. Lets reference real arxiv links in tests